src/FrontendAccountCreation.Core.UnitTests/AccountMapperTests.cs

```diff
@@ -14,7 +14,7 @@ namespace FrontendAccountCreation.Core.UnitTests;
 public class AccountMapperTests
 {
     [TestMethod]
-    public void testCreateAccountModel()
+    public void CreateAccountModel()
```

Comment:
Can you use Arrange Act Assert comments in the tests. Its hard to see where the act is :)

Reply:
Done